doc/changelog/11-infrastructure-and-dependencies/14189-configure+rewrite_mod.rst

```diff
@@ -0,0 +1,21 @@
+- **Changed:**
+  Coq's ``./configure`` script has gone thru a major cleanup. In
+  particular, the following options have been removed:
+  - ``-force-caml-version``, ``-force-findlib-version``: Coq won't
+  compile with OCaml or findlib lower than the required versions;
+  - ``-vmbyteflags``, ``-custom``, ``-no-custom``: linking options for
+  toplevels are now controlled in ``topbin/dune``;
+  - ``-ocamlfind``: Coq will now use the toolchain specified in the
+  Dune configuration; this can be controlled using the workspaces
+  feature;
+  - ``-nodebug``: Coq will now follow the standard, which is to always
+  pass ``-g`` to OCaml; this can be modified using a custom Dune workspace;
+  - ``-flambda-opts``: compilation options are now set in Coq's root
+  ``dune`` file, can be updated using a custom Dune workspace;
+  - ``-local``, ``-bindir``, ``-coqdocdir``, ``-annotate``,
+  ``-camldir``, ``-profiling``: these flags were deprecated in 8.14,
+  and are now removed.
```

Comment:
```suggestion
  compile with OCaml or findlib lower than the required versions;
  - ``-vmbyteflags``, ``-custom``, ``-no-custom``: linking options for
  toplevels are now controlled in ``topbin/dune``
  - ``-ocamlfind``: Coq will now use the toolchain specified in the
  Dune configuration; this can be controlled using the workspaces
  feature
  - ``-nodebug``: Coq will now follow the standard, which is to always
  pass ``-g`` to OCaml; this can be modified using a custom Dune workspace
  - ``-flambda-opts``: compilation options are now set in Coq's root
  ``dune`` file, can be updated using a custom Dune workspace
  - ``-local``, ``-bindir``, ``-coqdocdir``, ``-annotate``,
  ``-camldir``, ``-profiling``: these flags were deprecated in 8.14,
  and are now removed
```

Don't need ; for bulleted items

Reply:
That's a tricky one, for example, see
> If you’re trying to figure out how to punctuate a list, the best place to start is your style manual. Preferences vary from guide to guide. For example, the European Union’s English Style Guide suggests:
>
> - a comma or no punctuation for lists that do not contain the main verb
> - a full stop after each sentence for lists that contain at least one item with multiple sentences
> - a semicolon otherwise.

Reply:
But you left out the next sentence, which IMHO is much more common.  (It's also a bit curious to cite a style guide generated by the EU.  I doubt the British would consider that an authoritative guide.  I think it's probably intended for documents generated by EU bureaucrats so it's more fussy and different to some degree from standard usage.)

> On the other hand, the Australian Style Manual (Wiley, Sixth Edition) suggests no punctuation unless there is a full sentence (or multiple sentences), in which case there should be a full stop.

Reply:
I am not knowledgeable on this matter, but in many of my scientific papers, they require me to adhere to this, so I dunno.

I personally prefer the punctuation-free style, but I have been forced to correct it so many times that I wonder what is the right choice here.

Reply:
That's what I wrote "that's tricky", because so far I don't find a lot of consensus. Personally I don't care at all.

Reply:
Don't worry about it.